audformat/core/scheme.py

```diff
@@ -172,6 +172,21 @@ def __init__(
         self.maximum = maximum if self.is_numeric else None
         r"""Maximum value"""
 
+    @property
+    def is_numeric(self) -> bool:
+        r"""Data type is numeric.
+
+        Returns:
+            ``True`` if data type is numeric
+
+        """
+        return self.dtype in (define.DataType.INTEGER, define.DataType.FLOAT)
+
+    @property
+    def labels_as_list(self) -> typing.List:
+        r"""Scheme labels as list"""
+        return self._labels_to_list(self.labels)
```

Comment:
I think we need to handle the case where the scheme has no labels, e.g.:

```python
>>> audformat.Scheme().labels_as_list
TypeError: 'NoneType' object is not iterable
```

Maybe return an empty list in that case?

Reply:
Yes, good point.

I now return an empty list and also added a test for that case.